src/nginx/module.cc

```diff
@@ -239,20 +262,31 @@ ngx_command_t ngx_esp_commands[] = {
         //
         ngx_string("grpc_pass"),
         NGX_HTTP_LOC_CONF | NGX_CONF_NOARGS | NGX_CONF_TAKE12,
-        ConfigureGrpcBackendHandler, NGX_HTTP_LOC_CONF_OFFSET, 0, nullptr,
+        ConfigureGrpcBackendHandler,
```

Comment:
Please run scrip

script/check-style

to check your style

Reply:
Done.  Actually it is script/robot-style.